src/schemas/json/jsconfig.json

```diff
@@ -33,7 +33,7 @@
     "includeDefition": {
       "properties": {
         "include": {
-          "description": "Specifies a list of glob patterns that match files to be included in compilation. If no 'files' or 'include' property is present in a tsconfig.json, the compiler defaults to including all files in the containing directory and subdirectories except those specified by 'exclude'.",
+          "description": "Required TypeScript verion 2.0 or greater. Specifies a list of glob patterns that match files to be included in compilation. If no 'files' or 'include' property is present in a tsconfig.json, the compiler defaults to including all files in the containing directory and subdirectories except those specified by 'exclude'.",
```

Comment:
`Requires` instead of `Required`. also i would make this the last sentence in the documentation. this also applies to the rest of the file.